runtime/queries/htmldjango/indents.scm

```diff
@@ -0,0 +1,3 @@
+(paired_statement) @indent
+(end_paired_statement) @indent_end
+(branch_statement) @branch
```

Comment:
These look like they're in nvim style (we don't use `branch` or `indent_end` iirc). Could you adjust them to the helix ones? https://docs.helix-editor.com/guides/indent.html

I'm not caught up yet on the indentation mechanism so actually I'm not too sure how to these should look myself 😅

The default indent mechanism (removing this file) might work well. It seems to work pretty well for HTML for me

Reply:
I changed it but didn't go and test it out.